.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
editor specific functionality also shouldn't be committed, you can add the `.vscode/` directory to the `.gitignore`

Reply:
fixed